<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagementWidget</class>
 <widget class="QWidget" name="ManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="prevQuestion">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>← 上一題</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextQuestion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>600</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一題 →</string>
   </property>
  </widget>
  <widget class="QLabel" name="corrCount">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="progress">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="optionWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1000</width>
     <height>465</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
